tests/unit-tests/course-video/blocks/test-class-sensei-course-video-blocks-video-extension.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class Test_Sensei_Course_Video_Blocks_Video_Extension extends WP_UnitTestCase {
+	public function test_wrap_only_youtube_embed() {
```

Comment:
I think the name of this test is not correct after the refactor. :)
Also, maybe should we add some comments here to describe it, similar to the other tests?